pages/design-tokens/typesetting/overview.md

```diff
@@ -477,7 +477,7 @@ The individual design token sections go into this in more detail, but here's a s
 </div>
 
 ### Family, size, and line height together
-The `typeset()` mixin outlined below accepts the tokens listed as well as a special `null` value. If either of the three properties get `null` instead of a token, the system will use a default value from `$uswds-theme-typography`: either `$theme-body-font-family`, `$theme-body-font-size`, or `$theme-body-line-height` depending on the property passed the `null` value.
+The `typeset()` mixin outlined below accepts the tokens listed as well as a special `null` value. If either of the three properties get `null` instead of a token, the system will use a default value from [typography settings](/documentation/settings/#typography-settings): either `$theme-body-font-family`, `$theme-body-font-size`, or `$theme-body-line-height` depending on the property passed the `null` value.
```

Comment:
remove "outlined below" 
add "See this example:" after line 480

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/1866/commits/c4c9e3607312caae655e6a3579461d57537e5d16